xmtp_mls/src/groups/mod.rs

```diff
@@ -862,6 +864,91 @@ where
         Ok(())
     }
 
+    // If no changes, returns empty Vec
```

Comment:
Think we can remove these comments. I put them here mostly as a hint to some useful test cases.

Reply:
removed.